V2/serverless.yml

```diff
@@ -106,3 +123,37 @@ functions:
             - POST
           route: customers/
           authLevel: anonymous
+
+  get_customers:
+    handler: time_tracker/customers/interface.get_customers
+    events:
+      - http: true
+        x-azure-settings:
+          methods:
+            - GET
+          route: customers/{id:?}
+          authLevel: anonymous
+
+  update_customer:
+    handler: time_tracker/customers/interface.update_customer
+    events:
+      - http: true
+        x-azure-settings:
+          methods:
+            - PUT
+          route: customers/{id}
+          authLevel: anonymous
+
+  delete_customer:
+    handler: time_tracker/customers/interface.delete_customer
+    events:
+      - http: true
+        x-azure-settings:
+          methods:
+            - DELETE
+          route: customers/{id}
+          authLevel: anonymous
+
+#endregion Fin Customer
+
+#endregion Fin Funciones
```

Comment:
I think that if you are going to put comments to separate the functionalities, it should be clearer and in English, for example it could be `end customer section`